engine/pkg/logutil/log.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logutil
+
+import (
+	"github.com/pingcap/log"
+	libModel "github.com/pingcap/tiflow/engine/lib/model"
+	"github.com/pingcap/tiflow/engine/pkg/tenant"
+	"go.uber.org/zap"
+)
+
+const (
+	/// framework const lable
+	constFieldFrameworkKey   = "framework"
+	constFieldFrameworkValue = true
+
+	/// app const label
+	// constFieldTenantKey and constFieldProjectKey is used to recognize metric for tenant/project
+	constFieldTenantKey  = "tenant"
+	constFieldProjectKey = "project_id"
+	// constFieldJobKey is used to recognize jobs of the same job type
+	constFieldJobKey = "job_id"
+	// FieldWorkerKey is used to recognize workers of the same job
+	constFieldWorkerKey = "worker_id"
```

Comment:
Have a question here, what if business logs the same key, such as
`["worker test"] [tenant=tenant1] [project_id=proj1] [job_id=job1] [worker_id=worker1] [type=worker][worker_id=dm-worker-1]`

Reply:
Our log may be used  by Es, so even if we have the same key here, we still can scan the oriented log.